self_test/generic/eigensolver_test/Makefile.am

```diff
@@ -0,0 +1,25 @@
+#Include commands common to every Makefile.am
+include $(top_srcdir)/config/makefile_templates/demo_drivers
+
+# DO NOT NEED TO CHECK FOR MPI BECAUSE IF WE DO NOT HAVE MPI WE DO NOT
```

Comment:
But we should have parallel versions of these tests! I have some already (in a private repository). Can share; I'll add this to the evp-related issues.